config/settings.yml

```diff
@@ -99,6 +99,7 @@
     :miq_namespace: management-infra
   :ems_amazon:
     :disabled_regions: []
+    :additional_regions:
```

Comment:
This would allow it to have the same format as `ManageIQ::Providers::Amazon::Regions::REGIONS` so we could merge it without looping through the array, see https://github.com/ManageIQ/manageiq-providers-amazon/pull/89/files#diff-786c7b2d19b8663da84109af12cc8a96R75 and https://github.com/ManageIQ/manageiq-providers-amazon/pull/89#discussion_r90764172

If you think it would be better as an array I can switch it to that